modules/network-observability-RTT-overview.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-RTT-overview_{context}"]
+= Round-Trip Time
+You can use TCP handshake Round-Trip Time (RTT) to analyze network flows. You can use flow RTT captured from eBPF `flow_monitor` hookpoints to help with the following:
```

Comment:
that is not accurate, RTT is now using new `fentry/tcp_rcv_established` eBPF hook tp read `SRTT` from TCP socket instead of calculating

Reply:
Addressed. This is now: You can use RTT captured from fentry/tcp_rcv_established eBPF hookpoints to read SRTT from the TCP socket to help with the following: